packages/suite/src/support/messages.ts

```diff
@@ -2063,7 +2063,7 @@ export default defineMessages({
         id: 'TR_CLOSE_WINDOW',
     },
     TR_COIN_DISCOVERY_LOADER_DESCRIPTION: {
-        defaultMessage: 'Checking passphrase wallet for balances & transactions',
+        defaultMessage: 'Preparing your wallet…',
```

Comment:
will this change be enough?

This has already been translated through crowdin. I am not sure if crowdin has some mechanism to tell "hey, this string changed, we need to re-translate it"?

I am approving this, but asking just in case.